### ISSUE TITLE
Support new gemini version

### ISSUE THREAD

**Issue by w0rm** [Saulis/gemini-sauce#3]
This plugin is broken in the recent gemini, it seems that `config.browsers` is renamed to `config._browsers`, and per browser options are now inside `desiredCapabilities` attribute.

I tried to change the code, but still can't make sauce labs open my page:

``` js
_.forEach(gemini.config._browsers, function(browser) {
        browser.desiredCapabilities['tunnel-identifier'] = config.tunnelId;
      });
```

Also new gemini plugin support is broken, but for this I have created [a separate issue](https://github.com/gemini-testing/gemini/issues/224)

**Comment by Saulis** [comment]
Looks like the `gridUrl` parameter is loaded into each `browser` object in the latest `Gemini`.

I'll publish a fix asap, this should do the trick:

``` js
sauce.start(opts, function(config) {
      _.forEach(gemini.config._browsers, function(browser) {
        browser.gridUrl = config.gridUrl;
        browser.desiredCapabilities['tunnel-identifier'] = config.tunnelId;
      });

      deferred.resolve();
    });
```

**Comment by w0rm** [comment]
@Saulis you might want to hold this issue because gemini plugin support is broken at the moment

**Comment by Saulis** [comment]
@w0rm the issue seems to be fixed, (see gemini-testing/gemini#209 and gemini-testing/gemini#220), I'd expect a new release shortly.

**Comment by w0rm** [comment]
@Saulis awesome, they've released the fix. Looking forward for gemini-sauce to adopt the new configuration structure. If you don't mind, I will try to adopt gemini-sauce myself and open pull request.
